framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/scenario/builder/WorkbenchKieServerPersistentScenarioBuilder.java

```diff
@@ -90,4 +90,14 @@
      */
     WorkbenchKieServerPersistentScenarioBuilder withGitHooksDir(String dir);
 
+    /**
+     * KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE: If OpenShift integration of Business Central is turned on, setting this parameter to true  enables
+     * connection to KIE Server via an OpenShift internal Service endpoint.
+     * (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
+     *
+     * @param flag if true, it will use the internal service route.
+     * @return Builder
+     */
+    WorkbenchKieServerPersistentScenarioBuilder usePublicIpAddress();
```

Comment:
According to Evan's comment in https://issues.redhat.com/browse/RHPAM-2642, we should set this flag to ensure we're using the public service route (something we need to assert the server instance later on).